paper-tab.html

```diff
@@ -135,7 +147,12 @@
 
       _updateNoink: function() {
         this.noink = !!this.noink || !!this._parentNoink;
-      }
+      },
+
+      _focusBlurHandler: function(event) {
+        Polymer.IronControlState._focusBlurHandler.apply(this, arguments);
+        this.fire('paper-tab-' + event.type);
```

Comment:
Doesn't this duplicate what `IronControlState` already does? Why are we firing an extra event?